src/MetaCompilation.Analyzers/Core/MetaCompilation.Analyzers.csproj

```diff
@@ -2,8 +2,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard1.3</TargetFramework>
+    <!--
+      PackageId is used by Restore. If we set it to MetaCompilation.Analyzers,
+      Restore would conclude that there is a cyclic dependency between us and the MetaCompilation.Analyzers nuget package.
+    -->
+    <PackageId>*$(MSBuildProjectFullPath)*</PackageId>
```

Comment:
Ah, good to know this is tricking NuGet. Do we know the NuGet bug for the cyclic dependency issue?

Reply:
Looks like it's a won't fix: https://github.com/NuGet/Home/issues/936